application/test/mmul.cpp

```diff
@@ -0,0 +1,43 @@
+//
+// Created by shrijit on 6/9/20.
+//
+
+#define DOCTEST_CONFIG_IMPLEMENT_WITH_MAIN
```

Comment:
I'd recommend creating a header for doctest and including that instead. All you config and macros for testing can go in that

Reply:
Sounds good. I will need to make a common module for that where I can place all the common stuff. Will add this to my TODO.